pkgs/tools/cd-dvd/brasero/wrapper.nix

```diff
@@ -14,5 +14,8 @@ in symlinkJoin {
       --prefix LD_PRELOAD : ${lib.makeLibraryPath [ libdvdcss ]}/libdvdcss.so
   '';
 
-  inherit (brasero-original) meta;
+  meta = brasero-unwrapped.meta // {
+    # - cdrtools is non-redistributable when mixed with GPL code
```

Comment:
Can we remove cdrtools from the wrapper entirely, assuming the libburn backend is sufficient (this is a question, not a request for change)?

Independent of that, I'd get why linking might be problematic, but this simply adds cdrtools to $PATH, it doesn't link, neither static nor dynamically.

Reply:
The comment here is inaccurate. The problem is that `cdrtools` is itself a combination of GPL and CDDL code.

I agree that we should keep `libdvdcss` and just drop `cdrtools` if possible (most distributions don’t even package `cdrtools` and we probably shouldn’t bother either).

Reply:
Yeah, in that case cdrtools should just be marked as non-redistributable, unfree or whatever, and it'd propagate into here. But as I said, if brasero works fine without cdrtools, we can just drop it from here.

Reply:
Dropped this commit

Reply:
Looks like `cdrtools` is still there, though?

Reply:
Yeah, guess that's up for a future PR.

Reply:
Yeah